src/ResourceManager/ServiceBus/Commands.ServiceBus/help/New-AzureRmServiceBusRule.md

```diff
@@ -30,13 +38,36 @@ PS C:\> New-AzureRmServiceBusRule -ResourceGroup Default-ServiceBus-WestUS -Name
 
 The New-AzureRmServiceBusRule cmdlet creates a new rule for the specified Subscription.
 
+
+### Example 2
+```
+PS C:\> New-AzureRmServiceBusRule -ResourceGroup Default-ServiceBus-WestUS -Namespace SBExample1 -Topic SBTopic -Subscription SBSubscription -Name SBRule -SqlExpression "mysqlexpression='test'" -ActionSqlExpression "SET myAction='test'" -RequiresPreprocessing
+```
+
+The New-AzureRmServiceBusRule cmdlet creates a new rule for the specified Subscription with ActionFilter.
+
 ## PARAMETERS
 
+### -ActionSqlExpression
+Action SqlFillter Expression
+
+```yaml
+Type: String
+Parameter Sets: RuleActionPropertiesSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
 ### -DefaultProfile
-The credentials, account, tenant, and subscription used for communication with azure.
+The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
```

Comment:
Please rerun the Update-MarkdownHelp with the -UseFullTypeName flag (https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-a-single-markdown-file)

Reply:
resolved